### PR TITLE
Merge Milestone4 To Dev Branch

### PR DESCRIPTION
**Pull request by MephistoDevelop**
Features Added: 
-Calculate Module
-Operate Module
-Implement modules on App.js

### CODE REVIEW

File: src/components/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/* eslint-disable radix */
+/* eslint-disable import/extensions */
+import { Big } from 'big.js';
+
+const Operate = (nOne, nTwo, operation) => {
+  let Total = '';
```
Comment (Oghenebrume50) [inline src/components/logic/operate.js:6]: Create two variables with the Big function, one for the first number and the other for the second number.

File: src/components/logic/operate.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  if (operation === '+') {
+    Total = Big(parseInt(nOne) + parseInt(nTwo));
```
Comment (Oghenebrume50) [inline src/components/logic/operate.js]: parseInt is really not needed here, the Big function would still convert it to a number anyways

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
### STATUS: CHANGES REQUIRED 

Hi @MephistoDevelop good job on this implementation, :+1:  please make changes as outlined below and request for another review :slightly_smiling_face: 

**RAPHAEL NORIODE** :sunglasses:

File: src/components/logic/operate.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  }
+  if (operation === '/') {
+    Total = None / Ntwo;
+  }
+  if (operation === '*') {
+    Total = None * Ntwo;
+  }
+  if (operation === '%') {
+    Total = None % Ntwo;
+  }
```
Comment (azdrenymeri) [inline src/components/logic/operate.js]: When doing arithmetic calculation inside `operate` function you need to use the Big.js api and NOT the standard javascript operators`(+,-./.%,*)`
Checkout [Big.js docs](https://mikemcl.github.io/big.js/#) on how to use the library.
`hint:`
```javascript
// addition
const x = new Big(0.1)
const y = x.plus(0.2)            // '0.3'
Big(0.7).plus(x).plus(y)   // '1.1'
```

**Review by azdrenymeri — CHANGES_REQUESTED** [review]
### Hello @MephistoDevelop :wave: 
#### There is one thing left and we are done :muscle: 
- [x] You need to use the Big.js API functions when doing arithmetic calculations inside the `operate` function.
### Kindly review the comment and implement the changed then add it for another review.

File: src/components/logic/operate.js (hunk 3 above)
Comment (MephistoDevelop) [reply src/components/logic/operate.js]: Ok, i got it,  and fixed !!, thanks for the notice

**Review by bellom — APPROVED** [review]
## Hi @MephistoDevelop  :clap: 

**Great work making all the necessary corrections from the previous review.**

**Note:** Little update - You are using the incorrect npm module, the one that is required in the Milestone Pathwright is [this one](https://www.npmjs.com/package/big-js), big-js not big.js. Change that before merging to dev.


## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**